.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
```

Comment:
Never seen this before - will it be used for every new issue on the repo?

Reply:
You'll have the option to choose. I want to make it so that devs are the only ones that can open regular tickets, but not sure if thats possible. We can just inform users and the non-dev team to open tickets as feature requests, so they're visible to us on a separate board which we can occasionally prioritize/decide if we want to do that work